README.md

```diff
@@ -1,2 +1,2 @@
 # canvas-raycaster
-NOTE: This example repository has been archived. The example code and live example can now be found at [mdn/museum/canvas-raycaster](https://github.com/mdn/museum/tree/main/canvas-raycaster) and http://mdn.github.io/museum/canvas-raycaster/.
+> NOTE: This example repository has been archived. The example code and live example can now be found at [mdn/museum/canvas-raycaster](https://github.com/mdn/museum/tree/main/canvas-raycaster) and http://mdn.github.io/museum/canvas-raycaster/.
```

Comment:
The link at the end does not work yet. I will need to turn on gh-pages for the museum. Will let you know as soon as that is done.

Reply:
We are good to go here now.